MultimodalQnA/tests/test_compose_on_gaudi.sh

```diff
@@ -14,7 +14,9 @@ WORKPATH=$(dirname "$PWD")
 LOG_PATH="$WORKPATH/tests"
 ip_address=$(hostname -I | awk '{print $1}')
 
+export image_fn="apple.png"
 export video_fn="WeAreGoingOnBullrun.mp4"
+export caption_fn="apple.txt"
 
 function build_docker_images() {
     cd $WORKPATH/docker_image_build
```

Comment:
Please update the line 23 so that you are cloning the right repo/branch during tests and while this PR https://github.com/opea-project/GenAIComps/pull/852 still being merged.

`git clone https://github.com/opea-project/GenAIComps.git` should be replaced with:
```
git clone --single-branch --branch="melanie/mm-rag-enhanced" https://github.com/mhbuehler/GenAIComps.git
```